pages/data.js

```diff
@@ -30,26 +30,8 @@ export default class Explore extends React.Component {
   }
 
   componentDidMount() {
-    const { data, datasetNames } = this.props;
-    // In order to setup our filters object, we need to get each key, along with all unique records for that key.
-    // We can then create our filter object with all filters turned off by default
-    const recordKeys = Object.keys(data.records);
-
-    const filters = {};
-    recordKeys.forEach(key => {
-      filters[key] = Object.create(null, {});
-      const uniqueRecords = [...new Set(data.records[key])];
-      uniqueRecords.forEach(record => (filters[key][record] = true));
-    });
-
-    this.setState({
-      isLoading: false,
-      activeDataset: datasetNames[0],
-      data: {
-        [datasetNames[0]]: data,
-      },
-      filters,
-    });
+    const datasetNames = Object.keys(datasets);
+    this.fetchData(datasetNames[0]);
```

Comment:
We had a bunch of duplicated code for fetching `compressed` data and resetting filters. This DRYs things up a bit by calling `fetchData` directly from `componentDidMount` rather than trying to do the data fetching and filter resetting in both places.